pkg/kubelet/container/runtime.go

```diff
@@ -94,6 +94,8 @@ type Runtime interface {
 	RemoveImage(image ImageSpec) error
 	// Returns Image statistics.
 	ImageStats() (*ImageStats, error)
+	// Returns the filesystem path of the pod's network namespace
+	GetNetNS(containerID ContainerID) (string, error)
```

Comment:
I think it's cleaner to have a `NetworkPluggableRuntime` (name?) interface which just has `GetNetNS` since that's the only method needed, and then docker & rkt can implement it and you can cast to that, rather than a concrete implementation.
If some runtimes will stub out the method, it's not really part of a container runtime.